types/rox-browser/index.d.ts

```diff
@@ -1,60 +1,65 @@
-// Type definitions for rox-browser 4.8
+// Type definitions for rox-browser 5.0
 // Project: https://rollout.io
 // Definitions by: g-guirado <https://github.com/g-guirado>
+//                 rollout: <https://github.com/rollout>
```

Comment:
this needs to user name not org

Reply:
sorry about that, thought i saw some companies github account.

can we use the company link (not github), like there are in the 'react' definition for example?
or is it only github users?

Reply:
These are suppose to be authors on GitHub maintaining the types so we can ping them for reviewing changes .. so yeah GitHub user id is needed here